.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 #/avm/ptn/avd-lza/networking/ @Azure/avm-ptn-avd-lza-networking-module-owners-bicep @Azure/avm-module-reviewers-bicep
 #/avm/ptn/avd-lza/session-hosts/ @Azure/avm-ptn-avd-lza-sessionhosts-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/ptn/azd/aks/ @Azure/avm-ptn-azd-aks-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/ptn/azd/apim-api/ @Azure/avm-ptn-azd-apimapi-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
The latest commit is about aks, please check

Reply:
Looks good, thanks. Now we just have to ensure that (as per the [specs](https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships)) the team is created and added to the correct parent team. This will enable you to subsequently review & approve PRs on your own (unless you're the author, of course) 😄

Reply:
Seems alright now.